test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1262,16 +1263,15 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
     It "Validate Invoke-RestMethod -DisableKeepAlive" {
 
         # Operation options
-        $command = "Invoke-RestMethod -Uri 'http://httpbin.org/get' -TimeoutSec 5 -DisableKeepAlive"
+        $uri = Get-WebListenerUrl -Test 'Get'
+        $command = "Invoke-RestMethod -Uri '$uri' -TimeoutSec 5 -DisableKeepAlive"
 
         $result = ExecuteWebCommand -command $command
 
         # Validate response
-        $result.Output.headers.Host | Should Match "httpbin.org"
+        $result.Output.headers.Host | Should Match "localhost"
```

Comment:
shouldn't the be localhost... not just match?

Reply:
This is in a few locations

Reply:
it actually returns `localhost:<portnumber>` but, I didn't want to couple the port number which may change with the individual tests. If 8083 and 8084 will be the permanent ports, then this can be changes.

Reply:
I guess it could be

```powershell
$result.Output.headers.Host | Should Be $uri.Authority
```